single_kernel_mongo/core/abstract_upgrades.py

```diff
@@ -470,27 +457,9 @@ def are_all_units_ready_for_upgrade(self, unit_to_ignore: str = "") -> bool:
         return True
 
     @mongodb_only
-    def are_shards_status_ready_for_upgrade(self) -> bool:
```

Comment:
this was removed - as we perform more in depth status checks (i.e. can read/write), membership, etc.